src/Angor/Client/Storage/ClientStorage.cs

```diff
@@ -59,6 +59,23 @@ public void UpdateInvestmentProject(InvestorProject project)
         _storage.SetItem("projects", ret);
     }
 
+    public void SetInvestmentAmount(string projectId, decimal amount)
```

Comment:
ah I see this is a new field in storage, do we not have the inv amount specified already somewhere?